pkg/reconciler/v1alpha1/clusteringress/config/istio.go

```diff
@@ -38,6 +37,17 @@ const (
 	LocalGatewayKeyPrefix = "local-gateway."
 )
 
+var (
+	defaultGateway = Gateway{
```

Comment:
Can we add instructions somewhere describing the new way of adding new config entries?  Look like from now on we will need to make changes in the code to do this?  

I wonder if instead we should provide a set of "default" configmap which we can automatically patch post installation?  That would make it easier to add new config key.

Reply:
After I wrote this I don't know if that's even possible given what we want (default config won't be able to be migrated).

However you may know a way so I'll leave the comment here for you to decide.

Reply:
We need Operator docs on how to configure things in general.  I believe some of this is documented (e.g. custom domains), but the ConfigMaps are just one class of knobs here.